requirements.txt

```diff
@@ -0,0 +1 @@
+requests==2.31.0
```

Comment:
we should remove requests from being explicitly specified in requirements.txt and bundled with apps going forward per https://splunk.slack.com/archives/GBSGHUUD9/p1685364655311529

Reply:
> we should remove requests from being explicitly specified in requirements.txt and bundled with apps going forward per https://splunk.slack.com/archives/GBSGHUUD9/p1685364655311529

Removed the requests library from the requirements.txt
Now can you please approve it?